steps/cli/oc.go

```diff
@@ -168,4 +169,40 @@ var _ = gauge.Step("Configure GitHub token for git resolver in TektonConfig", fu
 var _ = gauge.Step("Configure the bundles resolver", func() {
 	patch_data := "{\"spec\":{\"pipeline\":{\"bundles-resolver-config\":{\"default-kind\":\"task\", \"defaut-service-account\":\"pipelines\"}}}}"
 	oc.UpdateTektonConfig(patch_data)
+})
+
+var _ = gauge.Step("Create signing-secrets for tekton chains", func(){
+	if os.Getenv("CHAINS_COSIGN_PUBLIC") == "" || os.Getenv("CHAINS_COSIGN_PRIVATE") == "" || os.Getenv("CHAINS_COSIGN_PASSWORD") == ""{
+		testsuit.T.Errorf("Cosign keys system variables were not exported")
+	}else {
+		if !oc.SecretExists("signing-secrets", "openshift-pipelines"){
+			chainsPublicKey := os.Getenv("CHAINS_COSIGN_PUBLIC")
+			chainsPrivateKey := os.Getenv("CHAINS_COSIGN_PRIVATE")
+			chainsPassword := os.Getenv("CHAINS_COSIGN_PASSWORD")
+			oc.CreateSigningSecretForTektonChains(chainsPublicKey, chainsPrivateKey, chainsPassword)
+		} else {
+			log.Printf("Secret \"signing-secrets\" already exists")
+		}
+	}
+})
+
+var _ = gauge.Step("Create quay secret for tekton chains", func(){
+	if os.Getenv("CHAINS_DOCKER_CONFIG_JSON") == "" || os.Getenv("CHAINS_CONFIG_JSON") == ""{
+		testsuit.T.Errorf("Robot credentials system variables were not exported")
```

Comment:
print the names of env variables in this message so that it's more user-friendly